freeipa/src/main/java/com/sequenceiq/freeipa/service/proxy/ProxyConfigService.java

```diff
@@ -37,6 +38,7 @@ public Map<String, SaltPillarProperties> createProxyPillarConfig(String environm
                 proxy.put("password", auth.getPassword());
             });
             proxy.put("noProxyHosts", config.getNoProxyHosts());
+            proxy.put("tunnel", stack.getTunnel());
```

Comment:
Last time I meant a central place where it is available in any case, even when there is no proxy present.